src/main/java/fact/io/hdureader/FITSStream.java

```diff
@@ -127,4 +127,8 @@ public Data readNext() throws Exception {
 
         return item;
     }
+
+    public Reader getReader() {
```

Comment:
Is this just for tests? Then Reader could be package private and the getter is unnecessary

Reply:
I need this also later in the `SamplePedestal` processor but I changed it to a skipToRow function as this is the only thing I need. No reason to expose the whole reader.